projects/objects/backgrounds/protos/TexturedBackground.proto

```diff
@@ -2,66 +2,100 @@
 # license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
 # license url: https://cyberbotics.com/webots_assets_license
 # tags: static
-# Background textured with a skybox.
+# Background textured with a background.
```

Comment:
is this correct or should it be
```suggestion
# Background textured with a skybox.
```

Reply:
I think this line would deserve a better description, the role of this PROTO has increased.